test/cppsuite/src/component/timestamp_manager.cpp

```diff
@@ -133,10 +133,26 @@ timestamp_manager::get_oldest_ts() const
 }
 
 wt_timestamp_t
-timestamp_manager::get_random_ts() const
+timestamp_manager::get_valid_read_ts() const
 {
+    /* Use get_oldest_ts here to convert from atomic to wt_timestamp_t. */
+    wt_timestamp_t current_oldest = get_oldest_ts();
+    wt_timestamp_t current_stable = _stable_ts;
+    if (current_stable > current_oldest) {
```

Comment:
Why is it possible that the current stable timestamp is larger than the oldest ts?

Reply:
The stable is meant to be newer than the oldest almost always.